addons/l10n_in/__init__.py

```diff
@@ -2,3 +2,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import models
+
+
+def init_settings(env):
+    """If the company is localized in India, activate the cash rounding by default.
+    """
+    # The cash rounding is activated by default only if the company is localized in India.
```

Comment:
The comment is confusing. It is not for **only** for Indian companies, but as soon as the module is installed, for **all** the companies.